2024-08-14 15:33:51,020 - werkzeug - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.0.17:5000
2024-08-14 15:33:51,021 - werkzeug - INFO - [33mPress CTRL+C to quit[0m
2024-08-14 15:34:15,070 - aiogram.dispatcher - INFO - Start polling
2024-08-14 15:34:15,196 - aiogram.dispatcher - INFO - Run polling for bot @EnvyApp_test_bot id=7247722923 - 'EnvyApp_test'
2024-08-14 15:34:15,632 - utils.api_requests - ERROR - HTTP error occurred: 404 Client Error: Not Found for url: https://envytelegramapp.onrender.com/api/v1/users/referral/None
2024-08-14 15:34:15,633 - utils.api_requests - INFO - Creating user with data: {'telegram_id': '506652203', 'telegram_username': 'koreechdhs', 'reffered_by': None}
2024-08-14 15:34:16,365 - utils.api_requests - ERROR - HTTP error occurred: 400 Client Error: Bad Request for url: https://envytelegramapp.onrender.com/api/v1/users/
2024-08-14 15:34:16,711 - utils.api_requests - ERROR - HTTP error occurred: 404 Client Error: Not Found for url: https://envytelegramapp.onrender.com/api/v1/users/referral/None
2024-08-14 15:34:16,712 - utils.api_requests - INFO - Creating user with data: {'telegram_id': '506652203', 'telegram_username': 'koreechdhs', 'reffered_by': None}
2024-08-14 15:34:17,400 - utils.api_requests - ERROR - HTTP error occurred: 400 Client Error: Bad Request for url: https://envytelegramapp.onrender.com/api/v1/users/
2024-08-14 15:34:17,750 - utils.api_requests - ERROR - HTTP error occurred: 404 Client Error: Not Found for url: https://envytelegramapp.onrender.com/api/v1/users/referral/None
2024-08-14 15:34:17,751 - utils.api_requests - INFO - Creating user with data: {'telegram_id': '506652203', 'telegram_username': 'koreechdhs', 'reffered_by': None}
2024-08-14 15:34:18,423 - utils.api_requests - ERROR - HTTP error occurred: 400 Client Error: Bad Request for url: https://envytelegramapp.onrender.com/api/v1/users/
2024-08-14 15:34:18,760 - utils.api_requests - ERROR - HTTP error occurred: 404 Client Error: Not Found for url: https://envytelegramapp.onrender.com/api/v1/users/referral/None
2024-08-14 15:34:18,761 - utils.api_requests - INFO - Creating user with data: {'telegram_id': '506652203', 'telegram_username': 'koreechdhs', 'reffered_by': None}
2024-08-14 15:34:19,494 - utils.api_requests - ERROR - HTTP error occurred: 400 Client Error: Bad Request for url: https://envytelegramapp.onrender.com/api/v1/users/
2024-08-14 15:34:19,610 - aiogram.event - INFO - Update id=168164775 is handled. Duration 3234 ms by bot id=7247722923
2024-08-14 15:34:19,638 - aiogram.event - INFO - Update id=168164774 is handled. Duration 4375 ms by bot id=7247722923
2024-08-14 15:34:19,670 - aiogram.event - INFO - Update id=168164776 is handled. Duration 2266 ms by bot id=7247722923
2024-08-14 15:34:19,701 - aiogram.event - INFO - Update id=168164777 is handled. Duration 1265 ms by bot id=7247722923
2024-08-14 15:35:54,366 - aiogram.dispatcher - INFO - Polling stopped
2024-08-14 15:35:54,621 - aiogram.dispatcher - INFO - Polling stopped for bot @EnvyApp_test_bot id=7247722923 - 'EnvyApp_test'
2024-08-14 15:35:57,348 - aiogram.dispatcher - INFO - Start polling
2024-08-14 15:35:57,472 - aiogram.dispatcher - INFO - Run polling for bot @EnvyApp_test_bot id=7247722923 - 'EnvyApp_test'
2024-08-14 15:36:42,020 - utils.api_requests - ERROR - HTTP error occurred: 404 Client Error: Not Found for url: https://envytelegramapp.onrender.com/api/v1/users/referral/None
2024-08-14 15:36:42,021 - utils.api_requests - INFO - Creating user with data: {'telegram_id': '5887302847', 'telegram_username': 'sun_4eeezzz', 'reffered_by': None}
2024-08-14 15:36:42,746 - utils.api_requests - INFO - User created successfully. Status code: 200
2024-08-14 15:36:42,880 - aiogram.event - INFO - Update id=168164778 is handled. Duration 1218 ms by bot id=7247722923
2024-08-14 15:40:12,935 - utils.api_requests - ERROR - HTTP error occurred: 404 Client Error: Not Found for url: https://envytelegramapp.onrender.com/api/v1/users/referral/None
2024-08-14 15:40:12,936 - utils.api_requests - INFO - Creating user with data: {'telegram_id': '506652203', 'telegram_username': 'koreechdhs', 'reffered_by': None}
2024-08-14 15:40:13,623 - utils.api_requests - ERROR - HTTP error occurred: 400 Client Error: Bad Request for url: https://envytelegramapp.onrender.com/api/v1/users/
2024-08-14 15:40:13,816 - aiogram.event - INFO - Update id=168164779 is handled. Duration 1297 ms by bot id=7247722923
2024-08-14 15:40:51,745 - utils.api_requests - ERROR - HTTP error occurred: 404 Client Error: Not Found for url: https://envytelegramapp.onrender.com/api/v1/users/referral/None
2024-08-14 15:40:51,746 - utils.api_requests - INFO - Creating user with data: {'telegram_id': '506652203', 'telegram_username': 'koreechdhs', 'reffered_by': None}
2024-08-14 15:40:52,427 - utils.api_requests - ERROR - HTTP error occurred: 400 Client Error: Bad Request for url: https://envytelegramapp.onrender.com/api/v1/users/
2024-08-14 15:40:52,560 - aiogram.event - INFO - Update id=168164780 is handled. Duration 1156 ms by bot id=7247722923
2024-08-14 15:57:43,055 - aiogram.dispatcher - INFO - Polling stopped
2024-08-14 15:57:43,306 - aiogram.dispatcher - INFO - Polling stopped for bot @EnvyApp_test_bot id=7247722923 - 'EnvyApp_test'
2024-08-14 15:57:46,778 - aiogram.dispatcher - INFO - Start polling
2024-08-14 15:57:46,907 - aiogram.dispatcher - INFO - Run polling for bot @EnvyApp_test_bot id=7247722923 - 'EnvyApp_test'
2024-08-14 15:58:18,785 - aiogram.dispatcher - INFO - Polling stopped
2024-08-14 15:58:19,041 - aiogram.dispatcher - INFO - Polling stopped for bot @EnvyApp_test_bot id=7247722923 - 'EnvyApp_test'
2024-08-14 15:58:21,487 - aiogram.dispatcher - INFO - Start polling
2024-08-14 15:58:21,628 - aiogram.dispatcher - INFO - Run polling for bot @EnvyApp_test_bot id=7247722923 - 'EnvyApp_test'
2024-08-14 15:58:23,702 - werkzeug - INFO - [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:5000
 * Running on http://192.168.0.17:5000
2024-08-14 15:58:23,703 - werkzeug - INFO - [33mPress CTRL+C to quit[0m
2024-08-14 15:59:05,397 - aiogram.dispatcher - INFO - Start polling
2024-08-14 15:59:05,502 - aiogram.dispatcher - INFO - Run polling for bot @EnvyApp_test_bot id=7247722923 - 'EnvyApp_test'
2024-08-14 15:59:23,943 - utils.api_requests - ERROR - HTTP error occurred: 404 Client Error: Not Found for url: https://envytelegramapp.onrender.com/api/v1/users/referral/None
2024-08-14 15:59:23,944 - utils.api_requests - INFO - Creating user with data: {'telegram_id': '506652203', 'telegram_username': 'koreechdhs', 'reffered_by': None}
2024-08-14 15:59:24,641 - utils.api_requests - ERROR - HTTP error occurred: 400 Client Error: Bad Request for url: https://envytelegramapp.onrender.com/api/v1/users/
2024-08-14 15:59:25,098 - utils.api_requests - ERROR - HTTP error occurred: 404 Client Error: Not Found for url: https://envytelegramapp.onrender.com/api/v1/users/referral/None
2024-08-14 15:59:25,099 - utils.api_requests - INFO - Creating user with data: {'telegram_id': '506652203', 'telegram_username': 'koreechdhs', 'reffered_by': None}
2024-08-14 15:59:25,793 - utils.api_requests - ERROR - HTTP error occurred: 400 Client Error: Bad Request for url: https://envytelegramapp.onrender.com/api/v1/users/
2024-08-14 15:59:27,143 - aiogram.event - INFO - Update id=168164781 is handled. Duration 21562 ms by bot id=7247722923
2024-08-14 15:59:28,463 - aiogram.event - INFO - Update id=168164782 is handled. Duration 3782 ms by bot id=7247722923
2024-08-14 15:59:35,924 - utils.api_requests - ERROR - HTTP error occurred: 404 Client Error: Not Found for url: https://envytelegramapp.onrender.com/api/v1/users/referral/None
2024-08-14 15:59:35,926 - utils.api_requests - INFO - Creating user with data: {'telegram_id': '506652203', 'telegram_username': 'koreechdhs', 'reffered_by': None}
2024-08-14 15:59:36,629 - utils.api_requests - ERROR - HTTP error occurred: 400 Client Error: Bad Request for url: https://envytelegramapp.onrender.com/api/v1/users/
2024-08-14 15:59:37,367 - aiogram.event - INFO - Update id=168164783 is handled. Duration 1780 ms by bot id=7247722923
2024-08-14 15:59:42,224 - aiogram.dispatcher - INFO - Polling stopped
2024-08-14 15:59:42,498 - aiogram.dispatcher - INFO - Polling stopped for bot @EnvyApp_test_bot id=7247722923 - 'EnvyApp_test'
2024-08-14 16:13:23,252 - aiogram.dispatcher - INFO - Start polling
2024-08-14 16:13:23,363 - aiogram.dispatcher - INFO - Run polling for bot @EnvyApp_test_bot id=7247722923 - 'EnvyApp_test'
2024-08-14 16:13:48,800 - utils.api_requests - ERROR - HTTP error occurred: 404 Client Error: Not Found for url: https://envytelegramapp.onrender.com/api/v1/users/referral/None
2024-08-14 16:13:48,801 - utils.api_requests - INFO - Creating user with data: {'telegram_id': '506652203', 'telegram_username': 'koreechdhs', 'reffered_by': None}
2024-08-14 16:13:49,495 - utils.api_requests - ERROR - HTTP error occurred: 400 Client Error: Bad Request for url: https://envytelegramapp.onrender.com/api/v1/users/
2024-08-14 16:13:49,656 - aiogram.event - INFO - Update id=168164785 is handled. Duration 1264 ms by bot id=7247722923
2024-08-14 16:20:39,330 - utils.api_requests - ERROR - HTTP error occurred: 404 Client Error: Not Found for url: https://envytelegramapp.onrender.com/api/v1/users/referral/None
2024-08-14 16:20:39,331 - utils.api_requests - INFO - Creating user with data: {'telegram_id': '506652203', 'telegram_username': 'koreechdhs', 'reffered_by': None}
2024-08-14 16:20:40,039 - utils.api_requests - ERROR - HTTP error occurred: 400 Client Error: Bad Request for url: https://envytelegramapp.onrender.com/api/v1/users/
2024-08-14 16:20:40,191 - aiogram.event - INFO - Update id=168164786 is handled. Duration 1266 ms by bot id=7247722923
2024-08-14 16:25:45,129 - utils.api_requests - ERROR - HTTP error occurred: 404 Client Error: Not Found for url: https://envytelegramapp.onrender.com/api/v1/users/referral/None
2024-08-14 16:25:45,130 - utils.api_requests - INFO - Creating user with data: {'telegram_id': '506652203', 'telegram_username': 'koreechdhs', 'reffered_by': None}
2024-08-14 16:25:45,849 - utils.api_requests - ERROR - HTTP error occurred: 400 Client Error: Bad Request for url: https://envytelegramapp.onrender.com/api/v1/users/
2024-08-14 16:25:45,970 - aiogram.event - INFO - Update id=168164787 is handled. Duration 1265 ms by bot id=7247722923
2024-08-14 16:28:53,277 - utils.api_requests - ERROR - HTTP error occurred: 404 Client Error: Not Found for url: https://envytelegramapp.onrender.com/api/v1/users/referral/None
2024-08-14 16:28:53,279 - utils.api_requests - INFO - Creating user with data: {'telegram_id': '506652203', 'telegram_username': 'koreechdhs', 'reffered_by': None}
2024-08-14 16:28:54,022 - utils.api_requests - ERROR - HTTP error occurred: 400 Client Error: Bad Request for url: https://envytelegramapp.onrender.com/api/v1/users/
2024-08-14 16:28:54,318 - aiogram.event - INFO - Update id=168164788 is handled. Duration 1437 ms by bot id=7247722923
2024-08-14 16:34:02,401 - utils.api_requests - INFO - User created successfully. Status code: 200
2024-08-14 16:34:02,402 - utils.api_requests - INFO - Creating user with data: {'telegram_id': '833773614', 'telegram_username': 'sxcllol', 'reffered_by': 11}
2024-08-14 16:34:03,108 - utils.api_requests - INFO - User created successfully. Status code: 200
2024-08-14 16:34:03,235 - aiogram.event - INFO - Update id=168164789 is handled. Duration 1187 ms by bot id=7247722923
2024-08-14 16:34:57,286 - utils.api_requests - INFO - User created successfully. Status code: 200
2024-08-14 16:34:57,287 - utils.api_requests - INFO - Creating user with data: {'telegram_id': '506652203', 'telegram_username': 'koreechdhs', 'reffered_by': 11}
2024-08-14 16:34:57,996 - utils.api_requests - ERROR - HTTP error occurred: 400 Client Error: Bad Request for url: https://envytelegramapp.onrender.com/api/v1/users/
2024-08-14 16:34:58,134 - aiogram.event - INFO - Update id=168164790 is handled. Duration 1187 ms by bot id=7247722923
2024-08-14 17:07:59,273 - utils.api_requests - ERROR - HTTP error occurred: 404 Client Error: Not Found for url: https://envytelegramapp.onrender.com/api/v1/users/referral/None
2024-08-14 17:07:59,274 - utils.api_requests - INFO - Creating user with data: {'telegram_id': '506652203', 'telegram_username': 'koreechdhs', 'reffered_by': None}
2024-08-14 17:08:00,442 - utils.api_requests - ERROR - HTTP error occurred: 400 Client Error: Bad Request for url: https://envytelegramapp.onrender.com/api/v1/users/
2024-08-14 17:08:00,445 - aiogram.dispatcher - ERROR - Failed to fetch updates - TelegramNetworkError: HTTP Client says - ServerDisconnectedError: Server disconnected
2024-08-14 17:08:00,446 - aiogram.dispatcher - WARNING - Sleep for 1.000000 seconds and try again... (tryings = 0, bot id = 7247722923)
2024-08-14 17:08:00,640 - aiogram.event - INFO - Update id=168164791 is handled. Duration 63985 ms by bot id=7247722923
2024-08-14 17:08:11,500 - aiogram.dispatcher - INFO - Connection established (tryings = 1, bot id = 7247722923)
2024-08-14 17:10:35,893 - utils.api_requests - ERROR - HTTP error occurred: 404 Client Error: Not Found for url: https://envytelegramapp.onrender.com/api/v1/users/referral/None
2024-08-14 17:10:35,894 - utils.api_requests - INFO - Creating user with data: {'telegram_id': '506652203', 'telegram_username': 'koreechdhs', 'reffered_by': None}
2024-08-14 17:10:36,679 - utils.api_requests - ERROR - HTTP error occurred: 400 Client Error: Bad Request for url: https://envytelegramapp.onrender.com/api/v1/users/
2024-08-14 17:10:36,867 - aiogram.event - INFO - Update id=168164792 is handled. Duration 1344 ms by bot id=7247722923
2024-08-14 17:10:44,285 - utils.api_requests - ERROR - HTTP error occurred: 404 Client Error: Not Found for url: https://envytelegramapp.onrender.com/api/v1/users/referral/None
2024-08-14 17:10:44,286 - utils.api_requests - INFO - Creating user with data: {'telegram_id': '506652203', 'telegram_username': 'koreechdhs', 'reffered_by': None}
2024-08-14 17:10:45,029 - utils.api_requests - ERROR - HTTP error occurred: 400 Client Error: Bad Request for url: https://envytelegramapp.onrender.com/api/v1/users/
2024-08-14 17:10:45,158 - aiogram.event - INFO - Update id=168164793 is handled. Duration 1264 ms by bot id=7247722923
2024-08-14 17:48:17,961 - utils.api_requests - ERROR - HTTP error occurred: 404 Client Error: Not Found for url: https://envytelegramapp.onrender.com/api/v1/users/referral/None
2024-08-14 17:48:17,964 - utils.api_requests - INFO - Creating user with data: {'telegram_id': '506652203', 'telegram_username': 'koreechdhs', 'reffered_by': None}
2024-08-14 17:48:18,756 - utils.api_requests - ERROR - HTTP error occurred: 400 Client Error: Bad Request for url: https://envytelegramapp.onrender.com/api/v1/users/
2024-08-14 17:48:19,043 - aiogram.event - INFO - Update id=168164794 is handled. Duration 53687 ms by bot id=7247722923
2024-08-14 18:49:06,845 - utils.api_requests - ERROR - HTTP error occurred: 404 Client Error: Not Found for url: https://envytelegramapp.onrender.com/api/v1/users/referral/None
2024-08-14 18:49:06,848 - utils.api_requests - INFO - Creating user with data: {'telegram_id': '506652203', 'telegram_username': 'koreechdhs', 'reffered_by': None}
2024-08-14 18:49:07,546 - utils.api_requests - ERROR - HTTP error occurred: 400 Client Error: Bad Request for url: https://envytelegramapp.onrender.com/api/v1/users/
2024-08-14 18:49:07,727 - aiogram.event - INFO - Update id=168164795 is handled. Duration 53484 ms by bot id=7247722923
2024-08-14 18:57:48,429 - utils.api_requests - ERROR - HTTP error occurred: 404 Client Error: Not Found for url: https://envytelegramapp.onrender.com/api/v1/users/referral/None
2024-08-14 18:57:48,430 - utils.api_requests - INFO - Creating user with data: {'telegram_id': '506652203', 'telegram_username': 'koreechdhs', 'reffered_by': None}
2024-08-14 18:57:49,175 - utils.api_requests - ERROR - HTTP error occurred: 400 Client Error: Bad Request for url: https://envytelegramapp.onrender.com/api/v1/users/
2024-08-14 18:57:49,347 - aiogram.event - INFO - Update id=168164796 is handled. Duration 1297 ms by bot id=7247722923
2024-08-14 19:36:56,137 - utils.api_requests - ERROR - HTTP error occurred: 404 Client Error: Not Found for url: https://envytelegramapp.onrender.com/api/v1/users/referral/None
2024-08-14 19:36:56,139 - utils.api_requests - INFO - Creating user with data: {'telegram_id': '506652203', 'telegram_username': 'koreechdhs', 'reffered_by': None}
2024-08-14 19:36:56,861 - utils.api_requests - ERROR - HTTP error occurred: 400 Client Error: Bad Request for url: https://envytelegramapp.onrender.com/api/v1/users/
2024-08-14 19:36:56,864 - aiogram.dispatcher - ERROR - Failed to fetch updates - TelegramNetworkError: HTTP Client says - Request timeout error
2024-08-14 19:36:56,864 - aiogram.dispatcher - WARNING - Sleep for 1.000000 seconds and try again... (tryings = 0, bot id = 7247722923)
2024-08-14 19:36:57,035 - aiogram.event - INFO - Update id=168164797 is handled. Duration 203485 ms by bot id=7247722923
2024-08-14 19:37:07,902 - aiogram.dispatcher - INFO - Connection established (tryings = 1, bot id = 7247722923)
2024-08-14 22:50:32,215 - utils.api_requests - ERROR - HTTP error occurred: 404 Client Error: Not Found for url: https://envytelegramapp.onrender.com/api/v1/users/referral/None
2024-08-14 22:50:32,219 - utils.api_requests - INFO - Creating user with data: {'telegram_id': '506652203', 'telegram_username': 'koreechdhs', 'reffered_by': None}
2024-08-14 22:50:32,945 - utils.api_requests - ERROR - HTTP error occurred: 400 Client Error: Bad Request for url: https://envytelegramapp.onrender.com/api/v1/users/
2024-08-14 22:50:32,949 - aiogram.dispatcher - ERROR - Failed to fetch updates - TelegramNetworkError: HTTP Client says - Request timeout error
2024-08-14 22:50:32,949 - aiogram.dispatcher - WARNING - Sleep for 1.000000 seconds and try again... (tryings = 0, bot id = 7247722923)
2024-08-14 22:50:33,134 - aiogram.event - INFO - Update id=168164798 is handled. Duration 303625 ms by bot id=7247722923
2024-08-14 22:50:44,000 - aiogram.dispatcher - INFO - Connection established (tryings = 1, bot id = 7247722923)
